build.wake

```diff
@@ -107,12 +107,17 @@ def doInstall dest kind =
     # kick off in parallel, but rsc and rsc_tool cannot be built at the same time right now.
     # Force them to be serial relative to each other.
     def install_rsc_bins =
+        require Pass rsc_migration = buildRSCMigration Unit
+
+        require Pass rsc_migration =
+            installAs "{dest}/bin/wake-remote-cache-migration" rsc_migration
```

Comment:
Tempted to name these `wake-rsc`, `wake-rsc-tool`, and  `wake-rsc-migration` instead if anyone has any opinions

Reply:
I think using `rsc` in the tool names is a good idea.